src/Viewer/renderers/mathInput.jsx

```diff
@@ -152,8 +155,9 @@ export default function MathInput(props) {
       // console.log(">>>", e.relatedTarget.id, checkWorkButton.props.id);
       setFocusedField(() => handleDefaultVirtualKeyboardClick);
       setFocusedFieldReturn(() => handleDefaultVirtualKeyboardReturn);
-      setFocusedFieldID(null);
+      // setFocusedFieldID(null);
```

Comment:
Removed this to style the focused mathInput-- but it seems important for the keyboard.